utils/conda-create-cudf.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export PYTHON_VERSION=3.6
+export NUMBA_VERSION=0.40.0
+export NUMPY_VERSION=1.14.5
+export PANDAS_VERSION=0.20.3
+export PYARROW_VERSION=0.10
+
+echo -e "\n"                                                                            
+echo "attempting to remove current conda environment cudf"                              
+conda-env   remove  --name cudf  --quiet --yes                                          
+echo "creating dev environment for cudf"                                                
+echo -e "\n"                                                                            
+conda       update  --name base --yes conda                                          && \
+conda       install --yes  python=$PYTHON_VERSION                                    && \
+conda       create  --name cudf  --yes python=$PYTHON_VERSION                        && \
+conda       install --name cudf  --yes --channel conda-forge                            \
+                                      --channel numba                                   \
+                                      --channel nvidia                                  \
+                                      nvstrings                                         \
```

Comment:
Typically the dependency list is specified (with minimum / maximum versions) in a .yml file.